### PR TITLE
Add workspaceRegistration flags to qa-brh

### PR DESCRIPTION
**Pull request by nss10**
Link to Jira ticket if there is one: [BRH-222](https://ctds-planx.atlassian.net/browse/BRH-222)

### Environments
* QA-BRH

### Description of changes
* Add all the flags and configs related to Workspace Registration

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa-brh.planx-pla.net/manifest.json
## :warning: Services on branch
- kayako-wrapper
- portal
- requestor
## :warning: Services are being downgraded
- portal

**Comment by PlanXCyborg** [comment]
# qa-brh.planx-pla.net/manifest.json
## :warning: Services on branch
- kayako-wrapper
- portal
- requestor
## :warning: Services are being downgraded
- portal

**Comment by PlanXCyborg** [comment]
# qa-brh.planx-pla.net/manifest.json
## :warning: Services on branch
- kayako-wrapper
- metadata
- portal
- requestor

**Review by jawadqur — DISMISSED** [review]

**Review by aartivnkt — APPROVED** [review]

**Comment by PlanXCyborg** [comment]
# qa-brh.planx-pla.net/manifest.json
## :warning: Services on branch
- kayako-wrapper
- metadata
- portal

**Comment by PlanXCyborg** [comment]
# qa-brh.planx-pla.net/manifest.json
## :warning: Services on branch
- kayako-wrapper
- metadata
- portal

**Review by aartivnkt — APPROVED** [review]

File: qa-brh.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -115,7 +115,8 @@
   "featureFlags": {
     "explorer": false,
     "discovery": true,
-    "discoveryUseAggMDS": true
+    "discoveryUseAggMDS": true,
```
[the remark this answers is not in the document]
Comment (nss10) [reply qa-brh.planx-pla.net/portal/gitops.json:118]: I'm sorry I don't follow! The flag for workspace Token Service is ` "discoveryUseAggWTS": true `

[the remark this answers is not in the document]
Comment (aartivnkt) [reply qa-brh.planx-pla.net/portal/gitops.json:118]: Same thing only -- I added that comment for heal folks who will look at this PR

[the remark this answers is not in the document]
Comment (aartivnkt) [reply qa-brh.planx-pla.net/portal/gitops.json:118]: Oh, yes gotcha Sai I see what you said now haha 🤦 Yes, WTS flag is `"discoveryUseAggWTS": true` not ` "discoveryUseAggMDS": true`. All letters appearing same to me right now

**Pull request merged**